ddsketch/mapping/index_mapping_test.go

```diff
@@ -85,6 +85,22 @@ func TestCubicallyInterpolatedMappingAccuracy(t *testing.T) {
 	}
 }
 
+func TestLowerBound(t *testing.T) {
+	testIndexes := []int{2, 10, 25, 100, 10000}
+	logMapping, _ := NewLogarithmicMapping(0.01)
+	linearMapping, _ := NewLinearlyInterpolatedMapping(0.01)
+	cubicalMapping, _ := NewCubicallyInterpolatedMapping(0.01)
+	for _, mapping := range []IndexMapping{logMapping, linearMapping, cubicalMapping} {
+		for _, i := range testIndexes {
+			lowerBound := mapping.LowerBound(i)
+			previous := mapping.Value(i-1)
+			next := mapping.Value(i)
+			assert.Greater(t, lowerBound, previous)
+			assert.Greater(t, next, lowerBound)
```

Comment:
As of now, I believe we can relax it and only require: `GreaterOrEqual` (which gives more flexibility when defining mappings).